Sources/Playlist/KPPlaylistController.swift

```diff
@@ -275,22 +275,25 @@ import PlayKit
             }
             
             let mediaOptions = self.prepareMediaOptions(forMediaEntry: currentEntry)
-            self.player?.setMediaAndUpdatePlugins(mediaEntry: currentEntry, mediaOptions: mediaOptions, pluginConfig: pluginConfig, callback: { error in
+            self.player?.setMediaAndUpdatePlugins(mediaEntry: currentEntry,
+                                                  mediaOptions: mediaOptions,
+                                                  pluginConfig: pluginConfig,
+                                                  callback: { error in
                 self.messageBus?.post(PlaylistEvent.PlaylistCurrentPlayingItemChanged())
             })
```

Comment:
We missed a weak self in this block.

```suggestion
            self.player?.setMediaAndUpdatePlugins(mediaEntry: currentEntry,
                                                  mediaOptions: mediaOptions,
                                                  pluginConfig: pluginConfig,
                                                  callback: { [weak self] error in
                self?.messageBus?.post(PlaylistEvent.PlaylistCurrentPlayingItemChanged())
            })
```